packages/polkit.rb

```diff
@@ -3,35 +3,42 @@
 class Polkit < Package
   description 'Application development toolkit for controlling system-wide privileges'
   homepage 'https://www.freedesktop.org/wiki/Software/polkit/'
-  version '0.120-7a99b'
+  version '0.120-a2bf5c'
   license 'LGPL-2'
   compatibility 'all'
-  source_url 'https://gitlab.freedesktop.org/xry111/polkit.git'
-  git_branch 'xry111/js91'
-  git_hashtag '7a99b67429b48357b36d29b3b650783af4b51ac1'
+  source_url 'https://gitlab.freedesktop.org/polkit/polkit.git'
+  git_hashtag 'a2bf5c9c83b6ae46cbd5c779d3055bff81ded683'
 
   binary_url({
-    aarch64: 'https://gitlab.com/api/v4/projects/26210301/packages/generic/polkit/0.120-7a99b_armv7l/polkit-0.120-7a99b-chromeos-armv7l.tpxz',
-     armv7l: 'https://gitlab.com/api/v4/projects/26210301/packages/generic/polkit/0.120-7a99b_armv7l/polkit-0.120-7a99b-chromeos-armv7l.tpxz',
-       i686: 'https://gitlab.com/api/v4/projects/26210301/packages/generic/polkit/0.120-7a99b_i686/polkit-0.120-7a99b-chromeos-i686.tpxz',
-     x86_64: 'https://gitlab.com/api/v4/projects/26210301/packages/generic/polkit/0.120-7a99b_x86_64/polkit-0.120-7a99b-chromeos-x86_64.tpxz'
+    aarch64: 'https://gitlab.com/api/v4/projects/26210301/packages/generic/polkit/0.120-a2bf5c_armv7l/polkit-0.120-a2bf5c-chromeos-armv7l.tpxz',
+     armv7l: 'https://gitlab.com/api/v4/projects/26210301/packages/generic/polkit/0.120-a2bf5c_armv7l/polkit-0.120-a2bf5c-chromeos-armv7l.tpxz',
+       i686: 'https://gitlab.com/api/v4/projects/26210301/packages/generic/polkit/0.120-a2bf5c_i686/polkit-0.120-a2bf5c-chromeos-i686.tpxz',
+     x86_64: 'https://gitlab.com/api/v4/projects/26210301/packages/generic/polkit/0.120-a2bf5c_x86_64/polkit-0.120-a2bf5c-chromeos-x86_64.tpxz'
   })
   binary_sha256({
-    aarch64: '41b4b9c4dc68b2f1960f571f09632422187fb3d726fccf90fe48e48539564e4c',
-     armv7l: '41b4b9c4dc68b2f1960f571f09632422187fb3d726fccf90fe48e48539564e4c',
-       i686: 'dc377359a5b4c77cd0fa429ff11134614b5c6766cb55dfbf85b4a3bcb29ba77c',
-     x86_64: 'ced0c15bbae3c494e287a217268fd5cb62e133d71a347b6d8224392f96c2d43f'
+    aarch64: '64f33f2ac0ab70d6aef5e1678bd83a5dbd9487ae50641402ffc5bd049409b4bf',
+     armv7l: '64f33f2ac0ab70d6aef5e1678bd83a5dbd9487ae50641402ffc5bd049409b4bf',
+       i686: 'd36368005afc56910018fbba79b8292aed5802db7fc8d9e6f593a3b81988292e',
+     x86_64: '7b775cbb16b44bde17a539955b33e14f97146ba10237ebeb01ec2ea700acafc7'
   })
 
-  depends_on 'js91'
+  depends_on 'duktape'
   depends_on 'elogind'
   depends_on 'gtk_doc' => :build
   depends_on 'gobject_introspection' => :build
 
+  def self.patch
+    # Fix meson 0.60+ compatibility
+    system 'curl -OLf https://gitlab.freedesktop.org/polkit/polkit/-/merge_requests/99.patch'
```

Comment:
Use `downloader.rb`?
```ruby
downloader('https://gitlab.freedesktop.org/polkit/polkit/-/merge_requests/99.patch')
```

Reply:
Might be easier to just put the patch in as a ruby here doc?

Reply:
done